xdsl/dialects/riscv.py

```diff
@@ -3016,6 +3026,65 @@ def assembly_line_args(self) -> tuple[AssemblyInstructionArg, ...]:
         return self.rd, self.rs1, self.rs2
 
 
+class RdRsRsFloatFloatIntegerOperationWithFastMath(RISCVInstruction, ABC):
```

Comment:
Can this not inherit from `RdRsRSFloatFloatIntegerOperation`? Looks like duplicated code

Reply:
Nope, because the inits get weird, I have a feeling that we should stick with this for now